### DISCUSSION TITLE
Moul lgrima

### DISCUSSION THREAD

**Discussion by k34n4y138** [geeksblabla/blanat#309] [General]
Thanks y'all for the competition, It was fun and and educational.
Apparently Twitter put messaging anyone outside your following list behind a paywall 😅

How to message the team?

**Comment by aboullaite** [comment]
Hello @k34n4y138, What is yout X handle ?

**Reply by k34n4y138** [reply]
Same as my GitHub
x.com/k34n4y138

**Reply by aboullaite** [reply]
DMd
